department-of-veterans-affairs/caseflow
# Intake Form Step 3: Confirmation - Linked Appeals & New Tasks

Resolves [Intake Form Step 3: Confirmation - Linked Appeals & New Tasks](https://jira.devops.va.gov/browse/APPEALS-29428)

# Description
Added table summary in step 3 to display information about tasks related to appeals.

## Acceptance Criteria

## Testing Plan
1. Navigate to Correspondence and select a batman veteran.
2. Add tasks related to an appeal, make sure you remember whether or not you select the 'evidence waived' option.
3. navigate to the confirmation page
4. You will see the tasks related to appeal summary at the bottom of the page. It will display the following

- A badge for the appeal type
- A hyperlink that will take you to the appeal
- Information about the task (currently active task, evidence window waived, assigned to)
- Edit Section hyperlink that will return the user to step 2
- Information about all additional tasks.


## User Facing Changes

 BEFORE|AFTER
 
<img width="1306" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110805785/432e08ac-484a-4495-82e5-fe5f6de95acd">
|
<img width="1251" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110805785/575c37a6-25da-401c-b9c9-0012f6ec4ce2">
